crt_portal/cts_forms/forms.py

```diff
@@ -1081,6 +1083,12 @@ class Meta:
                 'placeholder': 'Personal Description',
                 'aria-label': 'Personal Description'
             }),
+            'contact_email': EmailInput(attrs={
+                'class': 'usa-input',
+                'name': 'contact_email',
+                'placeholder': 'Email',
```

Comment:
Ahh missed this on first glance, let's have the placeholder be `Contact Email` to be consistent across the filter inputs.
```suggestion
                'placeholder': 'Contact Email',
```